SEED-platform/seed
# fix: set Bsyncr highlight to 'completed' in lieu of any other value

#### Any background context you want to provide?
See ticket

#### What's this PR do?
changes Bsynchr's highlight value from "Ooops..."  to "Completed." in lieu of having any other values to report.

#### How should this be manually tested?
Run a Bsyncr analysis and look at the highlight card.  It should be either on the property detail page or the property analyses page.  Make sure the value highlighted doesn't say "Oops" and instead says "Completed".

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2962

#### Screenshots (if appropriate)
